Backend/dev/migrations/dynamic-offer-driver-app/0285-dailyStats-table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS atlas_driver_offer_bpp.daily_stats (
+    id VARCHAR(36) NOT NULL PRIMARY KEY,
+    driver_id character(36) NOT NULL,
+    total_earnings INT,
+    number_of_rides INT,
+    total_distance INT,
+    of_date Date NOT NULL
```

Comment:
```
date
num_rides
```
looks better but your call
Also, backfilling required

Reply:
We already discussed about it and backfilling is not needed 
We will start storing and caching data from the day of release itself

Reply:
But why? Backfilling shouldn't be an issue. We can just write one query to fetch these details from ride table

Reply:
The whole thing was done for not to fetch data from ride table
We will be showing data in bar graphs for last 4 weeks, i.e 28 days and backfilling for each driver for last 28 days and calculating total for each day would've been an extra load. Hence we went for this.